docs-chef-io/content/inspec/resources/parse_config.md

```diff
@@ -53,9 +53,20 @@ where each test
 - May run a command to `stdout`, and then run the test against that output
 - May use options to define how configuration data is to be parsed
 
-## Matchers
+## Options
 
-For a full list of available matchers, please visit our [matchers page](/inspec/matchers/).
+This resource supports the following options for parsing configuration data. Use them in an `options` block stated outside of (and immediately before) the actual test:
+
+    options = {
+        assignment_regex: /^\s*([^:]*?)\s*:\s*(.*?)\s*$/,
+        multiple_values: true
+      }
+
+    output = command('some-command').stdout
```

Comment:
We should use this new example code; but bear in mind that we increasingly see issues with using Ruby variable assignment like this, because the code executes unconditionally even if the control is skipped/waived/excluded.

The only current solution is to write a custom resource that wraps up the code. In future we want InSpec to to properly skip code for excluded controls...